the_iron_yelp/restaurant.rb

```diff
@@ -0,0 +1,150 @@
+require 'json'
+
+WEEK_DAYS = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
+
+class Restaurant
+
+  attr_reader :id, :name, :address, :website, :hours, :prices, :cuisines, :ratings
+
+  #Instance Methods
+
+  def initialize(id, name, address, website, hours, prices, cuisines, ratings)
+
+    @id = id
+    @name = name
+    @address = address
+    @website = website 
+    @hours = hours 
+    @prices = prices 
+    @cuisines = cuisines 
+    @ratings = ratings
+
+  end
+
+  def to_s
+    "#{ name }"
```

Comment:
that's redundant - since name is already a string, its interpolated string value will be the same string.